src/widget.js

```diff
@@ -94,25 +94,29 @@ RemoteStorageWidget.prototype = {
     this.rs.on('connected', () => {
       console.debug('RS CONNECTED');
 
-      this.rs.sync.on('req-done', () => {
-        console.debug('SYNC REQ DONE');
-        this.rsSyncButton.classList.add('rs-rotate');
-      });
-      this.rs.sync.on('done', () => {
-        console.debug('SYNC DONE');
-        if (this.rsWidget.classList.contains('rs-state-unauthorized') ||
-            !this.rs.remote.online) {
-          this.updateLastSyncedOutput();
-        } else if (this.rs.remote.online) {
-          this.lastSynced = new Date();
-          console.debug('Set lastSynced to', this.lastSynced);
-          let subHeadlineEl = document.querySelector('.rs-box-connected .rs-sub-headline');
-          this.fadeOut(subHeadlineEl);
-          subHeadlineEl.innerHTML = 'Synced just now';
-          this.delayFadeIn(subHeadlineEl, 300);
-        }
-        this.rsSyncButton.classList.remove('rs-rotate');
-      });
+      if (this.rs.sync) {
```

Comment:
Either that or a type check. But using the existing feature flags would make sense, of course. Also, so that it still works when the underlying code is refactored and the private API changed.